meta-xt-domu/conf/machine/virtio-armv8-xt.conf

```diff
@@ -0,0 +1,12 @@
+#@TYPE: Machine
+#@NAME: virtio armv8 machine
+#@DESCRIPTION: Machine configuration for running a virtio armv8
+
+require conf/machine/include/arm/arch-arm64.inc
+
+KERNEL_IMAGETYPE = "Image"
+IMAGE_FSTYPES += " tar.bz2 ext4 wic.xz wic.bmap wic.xz.sha256sum"
```

Comment:
Do you really need WIC images? AFAIK, we do not use them. Rouge uses old good ext4 images.

Reply:
As we discussed we could use and not use them.
I said I implemented it with WIC and you said that's OK. So I leaved it.

Reply:
I wasn't expected that you'll add a new wic file. I don't think that we need it. This is extra bit of code that has no clear purpose, but we will need to support it.

Reply:
Done